boltz/base.go

```diff
@@ -339,3 +422,17 @@ func (entity *BaseExtEntity) UpdateBaseValues(ctx *PersistContext) {
 	ctx.Bucket.SetTimeP(FieldUpdatedAt, &now, nil)
 	ctx.Bucket.PutMap(FieldTags, entity.Tags, ctx.FieldChecker, false)
 }
+
+type NilEntity struct{}
```

Comment:
Is this to represent a zero value for entities? I assume that `nil` wasn't good enough? I don't see it used in the rest of this PR.

Reply:
I thought I'd removed that. It was to deal with some generics annoyance, but I got around it another way. I'll get rid of it.

Reply:
Removed